api/v1/Plan.php

```diff
@@ -1158,16 +1158,16 @@ private function GetBaseGeometryInformation(
         $baseInfo["geometry_id"] = $geometryId;
 
         $baseInfoQuery = $this->getDatabase()->query(
-            "SELECT geometry_id, geometry_persistent, geometry_mspid FROM geometry WHERE geometry_id = ?",
+            "SELECT geometry_id, geometry_persistent FROM geometry WHERE geometry_id = ?",
             array($geometryId)
         );
         $persistentId = $baseInfoQuery[0]["geometry_persistent"];
         $mspIdQuery = $this->getDatabase()->query(
-            "SELECT geometry_mspid FROM geometry WHERE geometry_persistent = ? AND geometry_mspid IS NOT NULL",
+            "SELECT IFNULL(geometry_mspid,0) as geometry_mspid FROM geometry WHERE geometry_persistent = ?",
```

Comment:
always return a geometry_mspid, even if it NULL in the database. Then it returns 0